test/e2e/operator/deployment_api.go

```diff
@@ -744,6 +776,84 @@ var _ = deploy.DescribeForSome("API", func(d *deploy.Deployment) bool {
 			}
 		})
 	})
+
+	Context("conversion", func() {
+		getAlphaDeployment := func(name string) alphaapi.Deployment {
+			return alphaapi.Deployment{
+				ObjectMeta: metav1.ObjectMeta{
+					Name: name,
+				},
+				Spec: alphaapi.DeploymentSpec{
+					DeploymentSpec: base.DeploymentSpec{
+						Image: dummyImage,
+					},
+				},
+			}
+		}
+		It("with default values", func() {
+			if d.HasOLM {
+				Skip("Conversion webhook is not supported by OLM")
+			}
```

Comment:
The conversion *webhook* may not be supported, but *conversion* is supported - it just works a bit differently.

In other words, don't skip this and the other tests below for OLM and instead just update the expectations where necessary.

Reply:
Yes, agree. I will modify the test.

Reply:
Done.